var.tfvars

```diff
@@ -39,6 +39,17 @@ dns_forwarders      = "8.8.8.8; 8.8.4.4"
 mount_etcd_ramdisk  = false
 installer_log_level = "info"
 ansible_extra_options = "-v"
+sysctl_tuned_options  = false
+#sysctl_options = ["kernel.shmmni = 16384", "net.ipv4.tcp_tw_reuse = 1"]
+#match_array = <<EOF
+#match_array:
```

Comment:
This line is needed? why user have to give the ansible var name?

Reply:
Moved back to the template file.